src/components/Garden/Filters/CompactFilter.js

```diff
@@ -171,7 +171,7 @@ function CompactFilterSlider({
             <SearchInput
               value={proposalNameFilter}
               onChange={onNameFilterChange}
-              placeholder="Search by name"
+              placeholder="Search by proposal name"
```

Comment:
We also need to update the `placeholder` of the `SearchInput` inside `CompactFilterBar` component